docs/modules/deployment/pages/core/containers.adoc

```diff
@@ -0,0 +1,94 @@
+
+[[container-deployment]]
+= Containerized Deployment
+:description: Learn how to install OpenNMS Horizon/Meridian in a containerized environment like Kubernetes or Red Hat OpenShift.
+
+If your organization uses Kubernetes or Red Hat OpenShift, OpenNMS makes a Helm chart available to simplify {page-component-title} deployment.
+For information on deploying in a Docker environment, see the Docker tabs in https://docs.opennms.com/horizon/latest/deployment/core/getting-started.html[installation and configuration].
+
+Each deployment through Helm charts has a single Core server, Grafana, a custom Ingress that shares the RRD files and some configuration files, and multiple Sentinels for flow processing.
+
+The OpenNMS Helm Chart is based and tested against the latest {page-component-title}.
+The minimum version available is
+ifeval::["{page-component-title}" == "Horizon"]
+Horizon 32.x.
+endif::[]
+
+ifeval::["{page-component-title}" == "Meridian"]
+Meridian 2023.x.
+endif::[]
+
+== Before you begin
+
+You must have a deep understanding of containerized environments (https://kubernetes.io/docs/home/[Kubernetes], https://access.redhat.com/documentation/en-us/openshift_container_platform/[Red Hat OpenShift]) and https://helm.sh/docs[Helm], the package manager for Kubernetes.
+
+You should also be familiar with {page-component-title}, its components, and how the environment works.
+
+== Requirements
+
+* https://kubernetes.io/docs/reference/kubectl/[kubectl]
+* https://kubernetes.io/docs/reference/kubectl/[Helm] version 3
+* Kubernetes 1.20+ OR Red Hat OpenShift 4.12+
+
+When using cloud resources, make sure you have https://learn.microsoft.com/en-us/cli/azure/reference-index?view=azure-cli-latest[az] for Azure or https://cloud.google.com/sdk/gcloud[gcloud] for Google Cloud.
```

Comment:
I think we can remove this; As this is going to depend on how the customer wants to manage their environment